frontend/views/containers/chatroom/MessageBase.vue

```diff
@@ -41,7 +41,7 @@
         )
 
         p.c-text(v-else-if='text')
-          template(v-for='(objText, index) in textObjects')
+          template(v-for='objText in textObjects')
```

Comment:
`key` attribute here (for`v-for`) seems to be missing.

Reply:
I only removed unused parameter `index` here. And also we could not use `:key` attribute here since the content is simple. But it's fine to add it too.

Reply:
```
[plugin vue] <template> cannot be keyed. Place the key on real elements instead.
```

I tried to add `key` attribute to the template element and got this error.

Reply:
Try a `<div>`?